source/slang/slang-emit.cpp

```diff
@@ -472,6 +474,15 @@ String emitEntryPoint(
             break;
         }
 
+        // If we have obfuscation strip
+        if (compileRequest->getLinkage()->m_obfuscateCode)
+        {
+            IRStripOptions options;
+            options.shouldStripNameHints = true;
+            options.stripSourceLocs = true;
+            stripFrontEndOnlyInstructions(irModule, options);
```

Comment:
It is worrying that we have to perform this step here, because at this point we are in the back-end and everything sensitive needs to have been stripped in the front-end for obfuscation to really work (since serialization and deserialization are before this point).

We should endeavor to find whatever path in the front-end is leading to information leaking through to this step, plug the hole there, and then remove this stripping step to confirm that things are working correctly.